docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
@MichaelHatherly BTW, @dickfeynman is an alternate (old) account of mine that I don't use these days. I didn't realize my comments were being made from that account! Sorry about the confusion; I'll have to figure out if I've messed up some login configuration.